resources/config/snemo/demonstrator/simulation/primary_events/1.3/documentation/flprimaries-1.3.rst

```diff
@@ -0,0 +1,527 @@
+
+     * ``"versatile_generator"``  : 
+        *Particle with monokinetic energy*
+
+        * Triggered variant : ``"if_versatile"``
+        * Group : ``"User"``
+
+
+   * Default value  : ``"electron.1MeV"``
+   
+
+2. Parameter ``"generator/if_flat_versatile/particle"`` :
+
+.. _primary_events-generator-if_flat_versatile-particle:
+
+
+   *The particle type*
+
+   * Full path: ``"primary_events:generator/if_flat_versatile/particle"``
+   * Model: ``"peg.generator.vspg.particle.PM"``
+   * Type: ``string`` 
+   * Mutability : ``variable``
+   * Variable mode  : ``enumeration``
+   * Supported string values:
+
+     * ``"alpha"`` 
+     * ``"electron"`` 
+     * ``"gamma"`` 
+     * ``"neutron"`` 
+     * ``"positron"`` 
+
+   * Default value  : ``"gamma"``
+   
+
+3. Parameter ``"generator/if_flat_versatile/energy_min"`` :
+
+.. _primary_events-generator-if_flat_versatile-energy_min:
+
+
+   *The particle min energy (flat)*
+
+   * Full path: ``"primary_events:generator/if_flat_versatile/energy_min"``
+   * Model: ``"peg.generator.vspg.energy.PM"``
+   * Type: ``real`` 
+   * Unit label : ``"energy"``
+   * Preferred unit : ``"keV"``
+   * Real precision : ``0 keV``
+   * Mutability : ``variable``
+   * Variable mode  : ``interval``
+   * Domain: ``[1 keV;10000 keV]``
+   * Default value  : ``1000 keV``
+   
+
+4. Parameter ``"generator/if_flat_versatile/energy_max"`` :
+
+.. _primary_events-generator-if_flat_versatile-energy_max:
+
+
+   *The particle max energy (flat)*
+
+   * Full path: ``"primary_events:generator/if_flat_versatile/energy_max"``
+   * Model: ``"peg.generator.vspg.energy.PM"``
+   * Type: ``real`` 
+   * Unit label : ``"energy"``
+   * Preferred unit : ``"keV"``
+   * Real precision : ``0 keV``
+   * Mutability : ``variable``
+   * Variable mode  : ``interval``
+   * Domain: ``[1 keV;10000 keV]``
+   * Default value  : ``1000 keV``
+   
+
+5. Parameter ``"generator/if_versatile/particle"`` :
+
+.. _primary_events-generator-if_versatile-particle:
+
+
+   *The particle type*
+
+   * Full path: ``"primary_events:generator/if_versatile/particle"``
+   * Model: ``"peg.generator.vspg.particle.PM"``
+   * Type: ``string`` 
+   * Mutability : ``variable``
+   * Variable mode  : ``enumeration``
+   * Supported string values:
+
+     * ``"alpha"`` 
+     * ``"electron"`` 
+     * ``"gamma"`` 
+     * ``"neutron"`` 
+     * ``"positron"`` 
+
+   * Default value  : ``"gamma"``
+   
+
+6. Parameter ``"generator/if_versatile/energy"`` :
+
+.. _primary_events-generator-if_versatile-energy:
+
+
+   *The particle energy (monokinetic)*
+
+   * Full path: ``"primary_events:generator/if_versatile/energy"``
+   * Model: ``"peg.generator.vspg.energy.PM"``
+   * Type: ``real`` 
+   * Unit label : ``"energy"``
+   * Preferred unit : ``"keV"``
+   * Real precision : ``0 keV``
+   * Mutability : ``variable``
+   * Variable mode  : ``interval``
+   * Domain: ``[1 keV;10000 keV]``
+   * Default value  : ``1000 keV``
+   
+
+
+
```

Comment:
As above, if this is a generated file, it should not be under version control, but built (otherwise it'll easily get out of sync). Also, docs should go into `documentation`.